cdk/domino_cdk/provisioners/vpc.py

```diff
@@ -88,23 +140,51 @@ def provision_vpc(self, name: str, vpc: config.VPC):
                 subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE),
             )
 
-    def provision_bastion(self, name: str, bastion: config.VPC.Bastion) -> None:
+        # TODO until https://github.com/aws/aws-cdk/issues/14194
+        for idx, subnet_id in enumerate(self.vpc.select_subnets(subnet_type=ec2.SubnetType.PUBLIC).subnet_ids):
+            name = f"DisableMapPublicIpLaunch-{idx}"
+            cr.AwsCustomResource(
```

Comment:
I think we might be heading towards a place where we should just rip out the single-stack option.